src/Tizen.NUI.Scene3D/src/public/Controls/Camera.cs

```diff
@@ -0,0 +1,731 @@
+/*
+ * Copyright(c) 2022 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+using System;
+using System.ComponentModel;
+using Tizen.NUI.BaseComponents;
+
+namespace Tizen.NUI.Scene3D
+{
+    /// <summary>
+    /// Enumeration for type determination of how the camera operates.
+    /// </summary>
+    /// This will be released at Tizen.NET API Level 10, so currently this would be used as inhouse API.
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public enum CameraType
+    {
+        /// <summary>
+        /// Camera orientation is taken from the CameraActor.
+        /// </summary>
+        FreeLook,
+        /// <summary>
+        /// Camera is oriented to always look at a target.
+        /// </summary>
+        LookAtTarget
+    }
+
+    /// <summary>
+    /// Enumeration for the projection modes.
+    /// </summary>
+    /// This will be released at Tizen.NET API Level 10, so currently this would be used as inhouse API.
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public enum ProjectionMode
+    {
+        /// <summary>
+        /// Distance causes foreshortening; objects further from the camera appear smaller.
+        /// </summary>
+        PerspectiveProjection,
```

Comment:
I think Projection can be dropped here as PropjectionMode.PerspectiveProjection looks tautology.

Reply:
Thank you I modified the namings.